source/deployment/cluster.rst

```diff
@@ -1,116 +1,126 @@
 
-- `Deployment Guide <https://docs.mattermost.com/deployment/cluster.html#id1>`_ - **Instructions to set up and maintain high availability on your Mattermost servers.**
- - `Initial Setup Guide for High Availability <https://docs.mattermost.com/deployment/cluster.html#id2>`_
- - `Adding a Server to the Cluster <https://docs.mattermost.com/deployment/cluster.html#id4>`_
- - `Removing a Server from the Cluster <https://docs.mattermost.com/deployment/cluster.html#id9>`_
+Each component must be properly sized
+  Upon failure of one component, the remaining application servers, database servers, and load balancers must be sized and configured to carry the full load of the system. If this requirement is not met, an outage of one component can result in an overload of the remaining components, causing a complete system outage.
 
-- `Configuration and Compatibility <https://docs.mattermost.com/deployment/cluster.html#id13>`_ - **Details on configuring your system for high availability.**
- - `Mattermost Server <https://docs.mattermost.com/deployment/cluster.html#mattermost-server-configuration>`_ - Configure the cluster settings for the Mattermost application server. 
- - `Proxy Server <https://docs.mattermost.com/deployment/cluster.html#proxy-server-configuration>`_ - Configure NGINX or another load balancer to proxy servers in the cluster.
- - `File Storage Location <https://docs.mattermost.com/deployment/cluster.html#file-storage-configuration>`_ - Configure the file storage location to be compatible with a high availability setup.
- - `Database <https://docs.mattermost.com/deployment/cluster.html#database-configuration>`_ - Size and deploy a multi-database configuration for high availability and scaling.
+Most updates can be applied without service interruption
+  You can apply most configuration changes and dot release security updates without interrupting service, provided that you update the system components in the correct sequence.
 
-- `Troubleshooting <https://docs.mattermost.com/deployment/cluster.html#id14>`_ - **Advice on troubleshooting your high availability setup.**
+In some cases, updates must be applied during a maintenance window
+  Changes to configuration settings that require a server restart, and updates that involve a change to the database schema require a short period of downtime. Downtime for a server restart is around 5 seconds, and for a database schema update, downtime can be up to 30 seconds.
 
------
+Outline of the process
+~~~~~~~~~~~~~~~~~~~~~~
 
+The following outline describes the procedure for updating a high availability system while maintaining continuous operation of the service.
+
+1. **Disconnect redundant servers** - Configure load balancers to consolidate requests to a single application server capable of supporting anticipated traffic during the update process. You might choose to run updates during low traffic periods or during a scheduled maintenance window.
```

Comment:
0/5 could this info be consolidated in the "Update guide" section, rather than having two areas talking about update instructions?